react/features/toolbox/components/native/OverflowMenu.tsx

```diff
@@ -140,6 +159,23 @@ class OverflowMenu extends PureComponent<IProps, IState> {
             }
         };
 
+        const buttons = [ {
```

Comment:
Why is this change necessary?

Reply:
This is just for testing. To see that UI wise, everything looks as expected. 
It will be removed and instead we will use const { customToolbarButtons } = state['features/base/config'];

Reply:
Ah okidoki!